eq_translations/survey_schema.py

```diff
@@ -128,6 +135,9 @@ def get_answer_pointers(self):
         """
         return find_pointers_to(self.schema, 'label')
 
+    def get_description_pointers(self):
+        return find_pointers_to(self.schema, 'description')[1:]
```

Comment:
Can you assume that this will always return a list of description pointers in order?

Reply:
yes, the underlying data structure is dictionary, which is ordered from python 3.6 on(eq-translations is 3.7.2)

Reply:
There's two things here. Firstly, a schema doesn't have to define a root description to be valid, so you cannot depend on it being present in the schema or it's position in a list. You're better off explicitly checking whether '/description' is present and returning a list without it.

Secondly, find_pointers_to doesn't make any claims about whether what it returns is ordered, so you probably shouldn't be depending on it being.

Reply:
maybe we should add "description" to required? Shouldn't every schema have description?

Reply:
Even if we did, my point about the unordered nature of the list still applies. I'd think making the changes I've suggested above would be easier than opening another PR.